app/views/course/lesson_plan/items/_personal_or_ref_time.html.slim

```diff
@@ -4,7 +4,7 @@
   span title=t('course.lesson_plan.items.fixed_desc')
     = fa_icon 'lock'
 =< format_datetime(effective_time[attribute], datetime_format) if effective_time[attribute]
-- if effective_time[attribute] != reference_time[attribute]
+- if (effective_time[attribute] != reference_time[attribute]) && (effective_time[attribute] != nil)
```

Comment:
Temporary fix when an item has effective time but no reference time. This fix prevents the page from crashing, but we won't show effective time. Yupei mentioned a fix in his other branch, so will leave it for now. Affects these rails pages: assessment index, assessment show, video show.